gamemodes/terrortown/entities/weapons/weapon_san_christmas_cannon.lua

```diff
@@ -60,7 +60,6 @@ function SWEP:PrimaryAttack()
 
         if SERVER then
             owner:SetNWString("SantaLoadedItem", "")
-            owner:SetNWBool("SantaHasAmmo", false)
 
             if random_presents then
```

Comment:
```suggestion
            if random_presents then
                owner:SetNWBool("SantaHasAmmo", false)
```
Need to keep the HasAmmo check for when random presents are enabled. Otherwise santa will be able to shoot infinite presents.